install/tools/ipa-compat-manage

```diff
@@ -100,7 +100,10 @@ def main():
         if dirman_password is None:
             sys.exit("Directory Manager password required")
 
-    api.bootstrap(context='cli', in_server=True, debug=options.debug)
+    api.bootstrap(context='cli',
+                  in_server=True,
+                  debug=options.debug,
+                  confdir=paths.ETC_IPA)
```

Comment:
NACK, the script works fine with IPA_CONFDIR. It does neither modify local file nor requires root permission to run.